OSeMOSYS_PuLP_HP.py

```diff
@@ -57,7 +57,6 @@
 
 inputFile = "Test_case_Input.xlsx"  # Update with actual filename
```

Comment:
Don't hard code file names into the code - use a command-line argument instead and provide access via an import statement and function call.